src/main/java/com/iemr/hwc/service/registrar/RegistrarServiceImpl.java

```diff
@@ -848,4 +841,51 @@ public int searchAndSubmitBeneficiaryToNurse(String requestOBJ) throws Exception
 		int i = commonBenStatusFlowServiceImpl.createBenFlowRecord(requestOBJ, null, null);
 		return i;
 	}
+
+	public String saveFingerprints(FingerPrintDTO comingRequest) {
+		String response = "";
+		M_UserTemp user = userRepo.getUserByUsername(comingRequest.getUserName());
+		if(user !=null){
+			UserBiometricsMapping userBiometricsMapping = new UserBiometricsMapping();
+			userBiometricsMapping.setUserID(user.getUserID());
+			userBiometricsMapping.setFirstName(user.getFirstName());
+			userBiometricsMapping.setLastName(user.getLastName());
+			userBiometricsMapping.setUserName(user.getUserName());
+			userBiometricsMapping.setCreatedBy(user.getUserName());
+
```

Comment:
We can avoid this string matching if we use a flat DTO. Please check once with Waris so that you two are on the same page regarding this . If this is the type of payload that he is sending then fine.

Reply:
Change done